internal/plugins/helm/v1/api.go

```diff
@@ -206,12 +203,6 @@ func (p *createAPISubcommand) InjectResource(res *resource.Resource) error {
 		return fmt.Errorf("multiple groups are not allowed by default, to enable multi-group set 'multigroup: true' in your PROJECT file")
 	}
 
-	// Selected CRD version must match existing CRD versions.
-	// nolint:staticcheck
-	if pluginutil.HasDifferentCRDVersion(p.config, p.resource.API.CRDVersion) {
-		return fmt.Errorf("only one CRD version can be used for all resources, cannot add %q", p.resource.API.CRDVersion)
```

Comment:
Context related to this change would be helpful.

Reply:
This was Deprecated, then removed in kubebuilder,

https://github.com/kubernetes-sigs/kubebuilder/blob/v3.15.1/pkg/plugin/util/helpers.go#L36

but it's probably better to just copy pasta the code for helm to not break anything.